src/runtime/storage/storage-provider-base.ts

```diff
@@ -15,14 +15,14 @@ import {ModelValue, SerializedModelEntry} from './crdt-collection-model.js';
 import {KeyBase} from './key-base.js';
 import {BigCollectionStore, CollectionStore, Store, VariableStore} from '../store.js';
 import {PropagatedException} from '../arc-exceptions.js';
-
+import {Dictionary, Consumer} from '../hot.js';
 
 enum EventKind {
   change = 'Change'
 }
 
 // tslint:disable-next-line: no-any
-type Callback = (v: {[index: string]: any}) => void;
+type Callback = Consumer<Dictionary<any>>;
```

Comment:
Can we namespace this callback further? Like a `KeywordArgumentCallback` (`KwargCallback` for short).

Reply:
Do you mean to rename Callback to KwargCallback or to create a specialization?

The request is unclear to me.